recipes/mpl_qt_viz/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "mpl_qt_viz" %}
+{% set version = "1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mpl_qt_viz-{{ version }}.tar.gz
+  sha256: 113bf8e996afb7c8a14089fc3308ff5369c705ebfd20e37cda96797700d9c46b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
```

Comment:
This is inconsistent with the setuptools metadata: https://github.com/nanthony21/mpl_qt_viz/blob/cceb1151c1076fe23f797fefcb4c0956ccee9306/setup.py#L37

Reply:
Thanks, that should be fixed now.